src/main/java/picard/vcf/MergeVcfs.java

```diff
@@ -54,14 +54,76 @@
 import java.util.List;
 
 /**
- * Combines multiple VCF files into a single file. Input files must be sorted by their contigs
- * and, within contigs, by start position. Throws IllegalArgumentException if the contig lists
- * are not present in the input files, are not identical or if the sample lists are not the
- * same; this class uses the GATK to merge headers, which may throw exceptions if the headers
- * cannot be merged. See VCFUtils.smartMergeHeaders for details.
- * <p/>
- * An index file is created for the output file by default. Using an output file name with a
- * ".gz" extension will create gzip-compressed output.
+ * <h2>Overview</h2>
+ * Combines multiple variant files into a single variant file.
+ *
+ * <h3>Inputs</h3>
+ *  <ul>
+ *      <li>one or more input VCF files in text uncompressed (name ending with ".vcf"), compressed text compressed format (name ending with ".vcf.gz")
+ *          or binary format BCF (name ending with ".bcf"),</li>
+ *      <li>and optionally a sequence dictionary file (typically name ending in .dict) if the input VCF do not contain a
+ *          complete contig list and if the output index is to be created (true by default).</li>
+ *  </ul>
+ *  <p>
+ *  The input variant files must:
+ *     <ul>
+ *         <li>if there samples, the sample list must be the same across all input files,</li>
+ *         <li>input file headers must be contain compatible declarations for common annotations (INFO, FORMAT fields) and filters.</li>
+ *         <li>and input files variant records must be sorted by their contig and position following the sequence dictionary provided
+ *         or the header contig list.</li>
+ *     </ul>
+ * </p>
+ * <p>
+ *     You can either directly list the input files one by one using the <code>INPUT</code> argument or enclose this
+ *     in a file with a name ending in ".list" and provide that one in the command line argument instead.
+ * </p>
+ *
+ * <h3>Outputs</h3>
+ * An unified variant file sorted by contig and the by position in the contig following the dictionary provided in the input.
+ * <p>Also an index file will be produced along-side the output variant file unless
+ * the user un-sets the <code>CREATE_INDEX</code> argument.
+ * </p>
+ * <p>The name of the index fill will depend on the output format; for uncompressed VCF and BCF the index file name appends
+ * ".idx" to the name of the output file whereas for compressed VCF the index file name appendix is ".tbi".</p>
+ * <p>
+ *     The format used for the output is determined by the output file as it is the case of the input files.
```

Comment:
This doesn't read well for me.  Rephrase.  How about, `the output file format will match the format of the input files.`?

Reply:
Nope, is the output file name that determines the format as it is the case for every single input file individually. I see how this can be confusing  though, So I will change it.